### ISSUE TITLE
Api forces to set statistics parameter to true

### ISSUE THREAD

**Issue by n9niwas** [jdalrymple/gitbeaker#3376]
**Description**

- Node.js version: v20.2.0
- Gitbeaker version: 39.10.3
- Gitbeaker release (cli, node, browser, core, requester-utils): @gitbreaker/rest
- OS & version: macOS Monterey

**Steps to reproduce**

Try to use `Groups.allDescendantGroups` function:
```ts
// typescript errors
await new Gitlab({ token: '' }).Groups.allDescendantGroups('');
await new Gitlab({ token: '' }).Groups.allDescendantGroups('', {});

// works fine
await new Gitlab({ token: '' }).Groups.allDescendantGroups('', { statistics: true });
await new Gitlab({ token: '' }).Groups.allDescendantGroups('', { statistics: false as true });
```

**Expected behaviour**

I must be able to disable statistics in `allDescendantGroups`.

**Checklist**
- [x] I have checked that this is not a duplicate issue.
- [x] I have read the documentation.

**Comment by jdalrymple** [comment]
Hmm, ill give it a look!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`39.12.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/39.12.0) :rocket:
